Core/src/org/sleuthkit/autopsy/report/infrastructure/TableReportSettings.java

```diff
@@ -71,4 +85,17 @@
     List<String> getTagSelections() {
         return tagNames;
     }
+
+    boolean isUseCaseSpecificData() {
+        return useCaseSpecificData;
+    }
```

Comment:
@rcordovano The short answer is that there IS a way to make TableReportGenerator work without  persisting this flag (i.e. without having the flag be part of TableReportSettings). Basically we would have to make decision based on whether the list of artifact selections is populated or not. But I feel like this would make TableReportGenerator  way too aware of the internals how UI panels populate the lists. I think we should keep the flag. Perhaps we can rename it to something like "loadAllTagsAndArtifacts" or something like that.